### PR TITLE
database-diagram

### PR DESCRIPTION
**Pull request by brhanuhailu**
### In this pull request we;

- [X] Make a database schema diagram for the vet clinic database.
- [X] Add the diagram to repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,7 +160,7 @@ Feel free to check the [issues page](https://github.com/brhanuhailu/vet_clinic/i
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse
+I would like to thank Microverse and everyone who has contributed to this project
```
Comment (bobb-Rob) [inline README.md:163]: - [ ] In this milestone, you are expected to create a database according to the image below 👇🏾 

   image

Kindly create a file named - `schema_based_on_diagram.sql`, and use `CREATE TABLE` command to create the various tables, then use foreign and primary keys to implement the associations between them.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @brhanuhailu, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (brhanuhailu) [reply README.md:163]: @bobb-Rob  Thank you very much done

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    FOREIGN KEY (invoice_id) REFERENCES invoices(id) ON DELETE CASCADE,
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id) ON DELETE CASCADE
+);
+
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(255) NOT NULL,
+    name VARCHAR(255) NOT NULL
+);
+
```
Comment (t-yanick) [inline schema_based_on_diagram.sql:44]: - [ ] Create job working on implementing the changes. As per project requirements, `all "join table" for the many-to-many relastionships is created, even if they are not showed on the diagram.`
You should have something similar to the snippet below:
```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+CREATE INDEX ON invoices_items (invoice_id);
+CREATE INDEX ON invoices_items (treatment_id);
+CREATE INDEX ON invoices (medical_history_id);
+
```
Comment (t-yanick) [inline schema_based_on_diagram.sql:55]: - [ ] After creating the join table as above, you should create indexes for the foreign keys
 `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
 `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @brhanuhailu , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brhanuhailu** [comment]
Updated based on the give constructive feedback

File: schema_based_on_diagram.sql (hunk 3 above)
Comment (brhanuhailu) [reply schema_based_on_diagram.sql:55]: @t-yanick Thank you for constructive feedback

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**